web/admin/package.json

```diff
@@ -6,11 +6,16 @@
   "dependencies": {
     "@doubledutch/admin-client": "^1.2.0",
     "@doubledutch/firebase-connector": "^3.4.3",
+    "@doubledutch/react-components": "^0.0.8",
+    "@fortawesome/fontawesome": "^1.1.8",
```

Comment:
You should not need to add the fontawesome dependencies directly.  They get pulled in by react-components where needed